### PR TITLE
Update Quickstart

### PR DESCRIPTION
**Pull request by DmitryPR**

### CODE REVIEW

File: docs/Quickstart.md [hunk 1]
```diff
@@ -35,6 +35,15 @@ use_frameworks!
 target '<Your Target Name>' do
     pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle'
 end
+
+post_install do |installer|
```
Comment (ostap0207) [inline docs/Quickstart.md]: Some more info on why this change is needed + ticket would be a good help for the reviewer.

**Comment by DmitryPR** [comment]
!merge
